adoc/Release-Notes.adoc

```diff
@@ -33,6 +33,109 @@ endif::[]
+
+`env.DB_EXTERNAL_SSL_MODE` - this is set to `true` by default and is used by the database seeder to communicate over TLS to the external database. Set it to `false` to have the seeder communicate over plain TCP.
+
+`env.UAADB_TLS` - this is set to `enabled` by default and is used by UAA to communicate over TLS to the external database. Set it to `disabled` to skip TLS for UAA connections to the external database. Please check the external database section in the documentation for other usages of this flag.
+
+`env.CREDHUB_DB_REQUIRE_TLS` - this is set to `true` by default and is used by credhub to communicate over TLS to the external database. Set it to `false` to disable TLS connections to the external database server from credhub. 
+
+Note that the external database server *must* support both TLS and non-encrypted connections; if the external database server only accepts TLS connections some SCF components will not be able to communicate with the database server.
+
+* {product} 1.5.2 has been tested successfully with external database instances hosted on Amazon RDS MariaDB and Azure MariaDB.  
+
+* The `cf-usb` service broker is not compatible with an external database hosted on Azure. We recommend using the service brokers that are provided by {azure} for OSBAPI connections to their hosted database services. 
+
+* With {product} 1.5.2 if you are using external UAA with an external database, you must set up two separate database instances; one for UAA and one for SCF. One external database instance for both an external UAA and an SCF setup is not supported and will cause data conflicts resulting in deployment failures.
+
+* If you are upgrading {product} to 1.5.2 and use an external database in 1.5.1, please contact SUSE Technical Support on guidance with the help of the backup and restore cli.
+
+Please refer to the external database section of the documentation for more configuration options.
+
+
+====
+
+* If you are upgrading {product} to 1.5.2 and already use Minibroker to connect to external databases and are using {k8s} 1.16 or higher, which is the case with {caasp} 4.1, you will need to update the database version to a compatible version and migrate your data over via the database's suggested mechanism. This may require a database export/import.
+
+* Starting with {product} 1.5.2, you no longer need to set `UAA_CA_CERT` when using an external UAA with a certificate signed by a well known Certificate Authority (CA). It is only needed when you use an external UAA with either a certificate generated by the `secret-generator` or a self-signed certificate.
+
+* Prior to {product} 1.5.2, it's possible that rotating the CCDB secret keys may have been broken. This has been fixed with this version: please consult https://documentation.suse.com/suse-cap/1.5.2/html/cap-guides/cha-cap-ccdb-secret-rotation.html 
```

Comment:
It was just the documentation, that was broken, not the rotation itself. And it is not "possible", the documentation was definitely broken, with the single exception of an empty database, in which case the instructions would have worked (because there would be nothing to decrypt/re-encrypt).

Reply:
Updated accordingly to indicate the documentation has been corrected.